content/docs/reference/cmctl.md

```diff
@@ -317,7 +317,10 @@ cmctl x uninstall
 
 This command uninstalls any Helm-managed release of cert-manager.
 
-The CRDs will be deleted if you installed cert-manager with the option `--set CRDs=true`.
+Starting from cmctl `v2.0.0`, the uninstall command is safe and will not delete the CRDs
+by default, even if they were installed with the Helm chart (using the option `--set installCRDs=true`).
+
+> 👴 Before `v2.0.0`, cmctl would remove the CRDs if they were installed with the Helm chart (similar to Helm's behavior).
```

Comment:
Not sure what the old man icon is about :-) Typo?

Reply:
I thought that would be icon that indicates that this doc is for an old version.

Reply:
Maybe 🕐 could be used instead?

Reply:
Oh. Ok. If it was intentional, then I'm happy to go with that.

/lgtm
/hold because I think the 🕐 does look better.

Reply:
I updated the icon.